src/main/java/duke/task/DateTime.java

```diff
@@ -0,0 +1,61 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * DateTime class used to store date objects.
+ */
+public class DateTime {
```

Comment:
I really like how you abstracted out all logic regarding dateTime formatting to this Class! It even handles valid formatting!